src/libraries/Native/Unix/System.Native/pal_datetime.c

```diff
@@ -39,3 +41,18 @@ int64_t SystemNative_GetSystemTimeAsTicks()
     // in failure we return 00:00 01 January 1970 UTC (Unix epoch)
     return 0;
 }
+
+#if defined(TARGET_ANDROID)
+char* SystemNative_GetDefaultTimeZone()
+{
+    char defaulttimezone[PROP_VALUE_MAX];
+    if (__system_property_get("persist.sys.timezone", defaulttimezone))
+    {
+        return strdup(defaulttimezone);
```

Comment:
Who frees this memory?

Reply:
Since the DllImport method declaration returns a `string` this indicates to the marshaller that the C# side owns the memory and the GC will free it as usual.